lib/github_changelog_generator.rb

```diff
@@ -34,10 +34,14 @@ def initialize
     def run
       log = @generator.compound_changelog
 
-      output_filename = @options[:output].to_s
-      File.open(output_filename, "wb") { |file| file.write(log) }
-      puts "Done!"
-      puts "Generated log placed in #{Dir.pwd}/#{output_filename}"
+      if @options[:output]
```

Comment:
If the output is `""` this branch will be used. The empty string is not `false`.

Perhaps we can add a predicate method to the Options class, to be used here? Something like `#output_filename?` which would return true if a non-blank, non-`"-"` value was set.

Reply:
Impl `#write_to_file?` at c3c45e8

> which would return true if a non-blank, non-"-" value was set.

I tried some tests, but `--output -` were not handed over to `GitHubChangelogGenerator::ChangelogGenerator#run` .

# Example
```ruby
# lib/github_changelog_generator.rb 

    def run
      log = @generator.compound_changelog

      # debug log
      p @options[:output]
```

```bash
$ bundle exec ./bin/github_changelog_generator --user sue445 --project my_repo --output - --since-tag v0.12.0
(snip)
nil

$ bundle exec ./bin/github_changelog_generator --user sue445 --project my_repo --output "-" --since-tag v0.12.0
(snip)
nil

$ bundle exec ./bin/github_changelog_generator --user sue445 --project my_repo --output "" --since-tag v0.12.0
(snip)
""

$ bundle exec ./bin/github_changelog_generator --user sue445 --project my_repo --output --since-tag v0.12.0
(snip)
nil
```

I think the hyphen is interpreted as a delimiter

Reply:
@sue445 Oh! Such a trip-up - sorry to have led you down a wrong road.